property/renderers.py

```diff
@@ -32,7 +32,9 @@ def single_property_format(self, data):
                 'client_name': client.client_name,
                 'phone': client.phone,
                 'email': client.email,
-                'address': client.address
+                'address': client.address,
+                'admin_email': client.client_admin.email,
+                'admin_id': client.client_admin.id
```

Comment:
I can see you are getting the client admin's id so we can use it to check against the currently logged in user's id and then decide whether or not to show the edit or delete buttons for a specific property....... nice work @MBenedicte